### ISSUE TITLE
ExecuteProcessasUser when using OS in non-English language with -Wait issue

### ISSUE THREAD

**Issue by cribban** [PSAppDeployToolkit/PSAppDeployToolkit#96]
When using the ExecuteProcessAsUser function together with the Wait parameter, this only seems to work with English language MUI. This was tested on Windows 7 with Swedish language pack.
The function searches for Scheduled task with status of "Running" when in Swedish that is named "Körs". The same applies for other MUI packs with localised variants.

What I did was to rewrite part of the Wait function, to use a language neutral function instead using a COM object. Im not sure if this is the best method but it did the trick and works with both English and Swedish OS languages.

This is the code I replaced it with (the Wait function for ExecuteProcessasUser):

```
    If ($Wait) {
        Write-Log -Message "Waiting for the process launched by the scheduled task [$schTaskName] to complete execution (this may take some time)..." -Source ${CmdletName}
        Start-Sleep -Seconds 1
        $schedule = new-object -com("Schedule.Service")
        $schedule.connect()
        $tasks = $schedule.getfolder("\").gettasks(0)
        Foreach ($task in $Tasks) {
            If ($task.Name -eq $schTaskName) {
                 While ($task.state -eq 4) {
                 Start-Sleep -Seconds 5
                 }
                 #  Get the exit code from the process launched by the scheduled task
                 [int32]$executeProcessAsUserExitCode = $task.lasttaskresult
                 Write-Log -Message "Exit code from process launched by scheduled task [$executeProcessAsUserExitCode]." -Source ${CmdletName}
            }
        }
    }
```

**Comment by amotaz** [comment]
We will address this in the 4.0 release of the toolkit.

**Comment by DennisVH** [comment]
This will be fixed in the next release for Windows Vista and higher

**Comment by seanlillis** [comment]
This is fixed in v3.7.0.
